core/controllers/dashboard.py

```diff
@@ -166,6 +167,8 @@ def _get_intro_card_color(category):
 
             feedback_thread_analytics = feedback_services.get_thread_analytics(
                 exp_summary.id)
+            exp_stats = stats_services.get_exploration_stats(
+                exp_summary.id, 'all')
```

Comment:
Actually, my point was to reuse the existing constant. This is because the reason we use named constants at all is to avoid skew due to typos/changes in the raw strings -- so each raw string should really only be declared once in the entire codebase. (Then, if its value changes, it does so for all usages of it.)